tests/ui/specs/VirtualList/VirtualGridList/ElementSize/VirtualGridList-GT-28445-specs.js

```diff
@@ -0,0 +1,36 @@
+const Page = require('../VirtualGridListPage');
+
+describe('Change ItemSize', function () {
+	beforeEach(function () {
+		Page.open();
+	});
+
+	it('should change minWidth and minHeight [GT-28445]', function () {
+		const defaultItemWidth = Page.getItemSize().width;
+		const defaultItemHeight = Page.getItemSize().height;
+		// Step 3: Knobs > VirtualGridList  > ItemSize.minWidth > 800
+		Page.inputMinWidth.moveTo();
+		Page.spotlightSelect();
+		Page.backSpace();
+		Page.backSpace();
+		Page.backSpace();
+		Page.numPad(8);
+		Page.numPad(0);
+		Page.numPad(0);
+		Page.spotlightDown();
+		// Step 3 Verify: The width of items grow bigger.
+		expect(Page.getItemSize().width).to.be.above(defaultItemWidth);
```

Comment:
How about verifying the minWidth 

```suggestion
		// Step 3 Verify: The width of items should be bigger than 400 for FHD.
		expect(Page.getItemSize().width).to.be.above(400);
```

Reply:
Suggested change.